src/nvim/option.c

```diff
@@ -512,17 +525,8 @@ static void set_options_default(int opt_flags)
 static void set_string_default(OptIndex opt_idx, char *val, bool allocated)
```

Comment:
is this defining/initializing the default or "setting the option to its default" ?

Reply:
It's changing the default value of an option, so the former